HTML:CSS learning project/style.css

```diff
@@ -87,30 +86,65 @@ body,
   height: auto;
 }
 
-
 .title h1 {
   color: #fff;
   font-size: 60px;
 }
 
-.button{
+.button {
   position: absolute;
   top: 63%;
   left: 50%;
   transform: translate(-50%, -50%);
   text-shadow: 0 0 3px black;
 }
 
-.title-button{
+.title-button {
```

Comment:
If you implement similar class names better stick with consistent naming. This should be `button-title`. In that case, it'd be easier to find and maintain code if there will be even more buttons. F.E `button-small`, `button-header` and etc.